lizmap/lizmap-modules/localiz/install/install.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+* @package   lizmap
+* @subpackage localiz
+* @author    aeag
+* @copyright 2011 3liz
+* @link      http://3liz.com
+* @license    All rights reserved
+*/
+
+
+class localizModuleInstaller extends jInstallerModule {
+
+    function install() {
+        // installer le CSS
+        $cssFile = jApp::wwwPath('css/localiz.css');
+        if (! file_exists($cssFile)) {
+            $this->copyFile('css/localiz.css', $cssFile);
+        }
+        
+        // installer le JS
+        $jsFile = jApp::wwwPath('js/localiz.js');
+        if (! file_exists($jsFile)) {
+            $this->copyFile('js/localiz.js', $jsFile);
+        }
+
+        // conf DB
+        $localConfFile = jApp::configPath('localconfig.ini.php');
+        if (file_exists($localConfFile)) {
+            $ini = new jIniFileModifier($localConfFile);
+            $ini->setValue('db_driver', 'pgsql', 'localiz_plugin');
+            $ini->setValue('db_host', 'pgpool', 'localiz_plugin');
+            $ini->setValue('db_port', '5432', 'localiz_plugin');
+            $ini->setValue('db_database', 'refgeo2', 'localiz_plugin');
+            $ini->setValue('db_user', 'visu', 'localiz_plugin');
+            $ini->setValue('db_password', 'visu', 'localiz_plugin');
```

Comment:
It's better to put these parameters into the profiles.ini.php (and so you don't have to create a "virtual" profil into search(), and it is more performant). And you should verify that there isn't already these parameters, to not overwrite values changed by the user.

To create a such profil, use instead the method `declareDbProfile`:

```
$this->declareDbProfile('localiz_plugin', array('driver'=>'pgsql', 'host'=>'...',...));
```